pkgs/by-name/th/thud/package.nix

```diff
@@ -25,14 +26,16 @@ rustPlatform.buildRustPackage rec {
     install -Dm644 assets/thud.thumbnailer $out/share/thumbnailers/thud.thumbnailer
     substituteInPlace $out/share/thumbnailers/thud.thumbnailer --replace "thud" "$out/bin/thud"
 
-    installManPage man/thud.1
+    installManPage target/man/thud.1
 
     installShellCompletion --cmd thud \
-      --bash <(cat completions/thud.bash) \
-      --fish <(cat completions/thud.fish) \
-      --zsh <(cat completions/_thud)
+      --bash <(cat target/completions/thud.bash) \
+      --fish <(cat target/completions/thud.fish) \
+      --zsh <(cat target/completions/_thud)
   '';
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
Nitpick: Can you split this into another commit?

Order does not matter. E.g.
- `thud: add passthru.updateScript`
- `thud: 0.3.0 -> 0.3.1`

Reply:
Fixed